subprojects/core/src/main/java/org/gradle/groovy/scripts/internal/InitialPassStatementTransformer.java

```diff
@@ -123,8 +124,8 @@ private String makePluginManagementError(String failureMessage) {
     }
 
     @Override
-    public PluginRequests create() {
-        return pluginBlockMetadataExtractor.getPluginRequests();
+    public BuildScriptMetadata create() {
```

Comment:
❓ It seems `BuildScriptMetadata` is not really used anywhere, is it?

Reply:
Good catch, removed. 9a7901b6b3997d56d0da6a9c16c54dbe5baa0c94